variables.tf

```diff
@@ -46,7 +46,14 @@ variable "retention" {
   default     = 30
 }
 
+variable "provision_trigger" {
+  type        = bool
+  description = "Whether or not to create a lambda trigger from an SNS topic"
+  default     = "false"
+}
+
 variable "sns_topic_arn" {
```

Comment:
nitpicky thing: if we're changing the format of this variable to a list perhaps we should add an `s` on the end to indicate that it contains more than one arn.

Reply:
Will do.